src/os/bluestore/bluestore_tool.cc

```diff
@@ -251,7 +251,7 @@ int main(int argc, char **argv)
     ("deep", po::value<bool>(&fsck_deep), "deep fsck (read all data)")
     ("key,k", po::value<string>(&key), "label metadata key name")
     ("value,v", po::value<string>(&value), "label metadata value")
-    ("allocator", po::value<vector<string>>(&allocs_name), "allocator to inspect: 'block'/'bluefs-wal'/'bluefs-db'/'bluefs-slow'")
+    ("allocator", po::value<vector<string>>(&allocs_name), "allocator to inspect: 'block'/'bluefs-wal'/'bluefs-db'")
```

Comment:
i don't quite understand this.. but we still handles bluefs-slow when serving "bluefs files list" and in `BlueFS::mkfs()`, which in turn calls `BlueFS::_init_alloc()`.

Reply:
@tchaikov
We used to have 4 allocators: 
a) 1 for BlueStore block data, 
b) 1 for BlueFS db, 
c) 1 for BlueFs wal, 
d) 1 for BlueFS on block.
When BlueFS was low on space it requested allocation from a) and transferred this space to d) as "new space". 

Now we got rid of d) and BlueFS is allocating space from a) directly.

So, we got rid of 1 allocator, but reporting of disk space used by BlueFS files remained unchanged.